streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/base/handler/GlobalExceptionHandler.java

```diff
@@ -52,15 +52,17 @@ public class GlobalExceptionHandler {
   @ResponseStatus(HttpStatus.INTERNAL_SERVER_ERROR)
   public RestResponse handleException(Exception e) {
     log.info("Internal server error：", e);
-    return RestResponse.fail("internal server error: " + e.getMessage(), ResponseCode.CODE_FAIL);
+    return RestResponse.fail(
```

Comment:
We can implement a new method to make it easier to call the RestResponse.fail method. e.g:

RestResponse.fail(ResponseCode.CODE_FAIL, "internal server error: %s", e.getMessage());

Reply:
https://github.com/apache/incubator-streampark/pull/3122

Reply:
Thank you for your reminder. I will modify it later